src/utils/filterLinks.ts

```diff
@@ -0,0 +1,18 @@
+import { Link } from '@/types/Folder.types';
+
+function filterLinks(links?: Link[], keyword?: string) {
+  if (!links) return;
+  if (!keyword) return links;
+
+  const lowered = keyword.toLowerCase().split(' ').join('');
```

Comment:
소문자로 변환 및 공백 제거 처리 작업이 적용되었으므로 nomalize 라는 표현이 더 좋을 것 같네요 🤗
(또, 아래와 동일하게 공백제거를 정규식으로 진행하면 좋을 것 같습니다.)

```suggestion
  const normalizedKeyword = keyword.toLowerCase().replace(/\s+/g, '');
```